analysers/analyser_merge_charging_station_FR.py

```diff
@@ -20,33 +20,170 @@
 ##                                                                       ##
 ###########################################################################
 
+import unittest
 from modules.OsmoseTranslation import T_
 from .Analyser_Merge import Analyser_Merge_Point, Source, CSV, Load_XY, Conflate, Select, Mapping
+import re
 
 
+WIKIDATA_MAP = {
```

Comment:
Like other parts, please, keep it into the class.

Reply:
yes, moved